docs/config/config.xml

```diff
@@ -26,6 +26,11 @@
     </database>
 
     <CouchDB>
+        <dbName>%COUCH_DATABASE%</dbName>
+        <hostname>%COUCH_HOSTNAME%</hostname>
+        <port>%COUCH_PORT%</port>
+        <admin>%COUCH_USERNAME%</admin>
+        <adminpass>%COUCH_PASSWORD%</adminpass>
```

Comment:
@driusan so you want me to romove this or actually create a script to port this data into mysql ?